src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsConformingFractures.hpp

```diff
@@ -119,10 +119,26 @@ class SinglePhasePoromechanicsConformingFractures : public CoupledSolver< Single
 
   void initializePostInitialConditionsPostSubGroups() override final;
 
+  void outputConfigurationStatistics( DomainPartition const & domain ) const override final;
+
   /**@}*/
 
 private:
 
+  void assembleCellBasedContributions( real64 const time_n,
+                                       real64 const dt,
+                                       DomainPartition & domain,
+                                       DofManager const & dofManager,
+                                       CRSMatrixView< real64, globalIndex const > const & localMatrix,
+                                       arrayView1d< real64 > const & localRhs );
+
+  virtual void  assembleCouplingTerms( real64 const time_n,
+                                       real64 const dt,
+                                       DomainPartition const & domain,
+                                       DofManager const & dofManager,
+                                       CRSMatrixView< real64, globalIndex const > const & localMatrix,
+                                       arrayView1d< real64 > const & localRhs ) override final;
```

Comment:
```suggestion
  virtual void assembleCouplingTerms( real64 const time_n,
                                      real64 const dt,
                                      DomainPartition const & domain,
                                      DofManager const & dofManager,
                                      CRSMatrixView< real64, globalIndex const > const & localMatrix,
                                      arrayView1d< real64 > const & localRhs ) override final;
```

Reply:
You must see something different in your IDE coz I don't see any difference in mine between what's there and the suggestion.